samples/vaultapi/upsert.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright (c) 2022 Skyflow, Inc.
+*/
+package main
+
+import (
+	"fmt"
+
+	logger "github.com/skyflowapi/skyflow-go/commonutils/logwrapper"
+	saUtil "github.com/skyflowapi/skyflow-go/serviceaccount/util"
+	Skyflow "github.com/skyflowapi/skyflow-go/skyflow/client"
+	"github.com/skyflowapi/skyflow-go/skyflow/common"
+)
+
+var bearerToken = ""
+
+func GetToken() (string, error) {
+
+	filePath := "<file_path>"
+	if saUtil.IsExpired(bearerToken) {
+		newToken, err := saUtil.GenerateBearerToken(filePath)
+		if err != nil {
+			return "", err
+		} else {
+			bearerToken = newToken.AccessToken
+			return bearerToken, nil
+		}
+	}
+	return bearerToken, nil
+}
+
+func main() {
+
+	defer func() {
+		if err := recover(); err != nil {
+			fmt.Println("error : ", err)
+		}
+	}()
+	logger.SetLogLevel(logger.INFO) //set loglevel to INFO
```

Comment:
Update comment to sentence structure: Set loglevel to INFO.

Reply:
Done